qa/workunits/suites/fsx.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-git clone git://git.ceph.com/xfstests.git
+git clone https://git.ceph.com/xfstests.git
```

Comment:
Was this run through the suites yet?  This repo doesn't seem to exist -- it's now named xfstests-dev.git.

Reply:
Fixed it.